elliottlib/cli/__main__.py

```diff
@@ -96,8 +96,12 @@
 @click.option("--id", metavar='BUGID', default=[],
               multiple=True, required=True,
```

Comment:
can no longer be required
```suggestion
              multiple=True, required=False,
```